products/samsungmobile.md

```diff
@@ -265,23 +247,41 @@ releases:
     eol: false
     link: https://doc.samsungmobile.com/SM-X706B/SER/doc.html
 
--   releaseCycle: "Galaxy S21 FE 5G"
-    releaseDate: 2022-01-07
+-   releaseCycle: "Galaxy S22 Ultra 5G"
+    releaseDate: 2022-02-25
+    support: true
+    eol: 2027-02-25 # "five years of security updates" (https://security.samsungmobile.com/securityPost.smsb)
+    link: https://doc.samsungmobile.com/SM-S908E/VAU/doc.html
+
+-   releaseCycle: "Galaxy S22+ 5G"
+    releaseDate: 2022-02-25
+    support: true
+    eol: 2027-02-25 # "five years of security updates" (https://security.samsungmobile.com/securityPost.smsb)
+    link: https://doc.samsungmobile.com/SM-S906E/XXV/doc.html
+
+-   releaseCycle: "Galaxy S22 5G"
+    releaseDate: 2022-02-25
+    support: true
+    eol: 2027-02-25 # "five years of security updates" (https://security.samsungmobile.com/securityPost.smsb)
```

Comment:
The link says "up to five years of security updates", which is not very precise, so I am not sure we should set this date. @Evernow any opinion on this ?

Reply:
I agree that the wording is weird, but it's been widely reported and understood to mean five years of updates (but no more than that).

For example, if you look at the context of https://news.samsung.com/us/samsung-galaxy-os-upgrade-one-ui-android-unpacked-2022/, they promise "up to four generations of One UI and Android OS upgrades on select devices", which they say "builds upon Samsung’s previous promise to provide three years of upgrades". That wouldn't be an improvement if "up to four" really still meant three.

Reply:
My vote it to merge this one and change it if samsung drops its support before these dates

Reply:
Ok, let's apply this then.